www/src/css/lib/vanilla-overrides.scss

```diff
@@ -5,4 +5,50 @@
 // If any styles from vanilla framework need to be overridden, corresponding
 // SCSS rules should be put in this file (not mixed with other styles).
 
+// scss/modules/_header.scss
+.banner .nav-primary ul li:hover {
+  background-color:#f7f7f7;
+}
 
+.banner .nav-primary ul li a,
+.banner .nav-primary ul li a:link,
+.banner .nav-primary ul li a:visited {
+  background-color:transparent;
+  font-size:initial;
+  border-left:0;
+  padding:14px 12px 8px 12px;
+}
+
+.banner .nav-primary ul li a:hover,
+.banner .nav-primary ul li a:link:hover,
+.banner .nav-primary ul li a:visited:hover {
+  background-color:transparent;
+}
+
+@media screen and (max-width: 640px) {
```

Comment:
Good catch on using the SCSS variable here.

I hadn't considered the mobile-first approach to be honest but now that you point it out it makes sense. Subtle but logical. I'll update things accordingly as it would mean having fewer media queries and should be easier to reason about.